install/gcp/up.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -u
+set -x
+
+PROJECT_ID=${PROJECT_ID:=$(gcloud config get project)}
+REGION=${REGION:-us-central1}
+ZONE=${ZONE:=${REGION}-a}
+INSTALL_OPERATOR=no # set to yes if you want to install operator
+
+# Enable required services.
+gcloud services enable container.googleapis.com
+gcloud services enable artifactregistry.googleapis.com
+
+export CLUSTER_NAME=substratus
+gcloud container clusters create ${CLUSTER_NAME} --location ${REGION} \
```

Comment:
Looks like in some places we create, in some places we check then create if not exists... Perhaps we standardize on one or the other. Simplicity favors always creating, practically speaking though, it might be simpler for the user to be able to re-run the script on failure (idempotency). I am open to either.

Reply:
I've added if statements everywhere